README.md

```diff
@@ -0,0 +1,96 @@
+# Renku
+
+[![Discourse](https://img.shields.io/discourse/status?server=https%3A%2F%2Frenku.discourse.group)](https://renku.discourse.group/)
+[![Gitter](https://img.shields.io/gitter/room/SwissDataScienceCenter/renku)](https://gitter.im/SwissDataScienceCenter/renku)
+
+**Renku** is an open-source platform that connects the ecosystem of
+data, code, and compute to empower researchers to build collaborative
+communities. It is aimed at independent researchers and data scientists
+as well as labs, collaborations, and courses and workshops.
+
+Once a Renku project is configured, anyone using it can take advantage
+of convenient access to data and code within pre-configured
+computational sessions. Projects are assembled by piecing together:
+
+- **Data connectors**: connect virtually any cloud-based or on-premise
+data source, like S3 buckets, WebDav or SFTP servers. Once you create a
+data connector, it can be reused across all of your projects and even be
+made available to your group or community.
+
+- **Source code**: link the source code repositories you are used to, like
+GitHub or GitLab and easily work on it within your project sessions.
+
+- **Compute environments**: use one of our pre-configure environments with
+VSCode, RStudio or Jupyter inside or have Renku build an image to match
+your repository\'s requirements. Alternatively, bring your own docker
+image and share it easily with others to really make your code and data
+sing.
+
+> [!NOTE]  
+> Renku is currently being rewritten; the new version, Renku 2.0 will
+> replace the \"legacy\" functionality fully in late Spring 2025. All of
+> the material linked below refers to the new version.
+
+## Getting Started
+
+A public instance of **RenkuLab** is available at
+https://renkulab.io/v2. To start exploring Renku, feel free to make an
+account and try it out! You can follow the [hands-on
+tutorial](https://renku.notion.site/Renku-2-0-Tutorials-1460df2efafc80c2b27acd221aa34a24?p=1a50df2efafc800f8554e30fd7458fa6&pm=s)
```

Comment:
This link opens the tutorial as a side panel. Is this intended?

Reply:
I know I saw that - I don't think it's a problem?

Reply:
It confused me at least. I would rather have the full page.